### ISSUE TITLE
Help requested: lintr error "The code depends on uninstalled packages"

### ISSUE THREAD

**Issue by wlangera** [inbo/checklist#171]
Hi

I do not understand the checklist issue for this pull request.
https://github.com/inbo/mbag-mas/pull/52

The warning in the logs [here](https://github.com/inbo/mbag-mas/actions/runs/20342843622/job/58446785930?pr=52#step:3:3184):
```
Warning: file=source/markdown/exploratie_duinen_weidestreek.Rmd,line=37,col=1,[object_usage_linter]
Could not find exported symbols for package "GVI" in libraries '/usr/local/lib/R/site-library',
'/usr/local/lib/R/library' (there is no package called 'GVI'). This may lead to false positives.
```

But when I run `checklist::check_project()` locally:
lintr: 1 error
```
------------------------------------------------------------------------------------------------------------------------
 The code depends on the following uninstalled packages: clustermq
```

**Comment by ThierryO** [comment]
Your code requires or depends on the `GVI` package. `checklist` failed to install it. From the raw GHA logs

```
2025-12-18T16:06:47.0332229Z Warning message:
2025-12-18T16:06:47.0332590Z package 'GVI' is not available for this version of R
2025-12-18T16:06:47.0332933Z 
2025-12-18T16:06:47.0333248Z A version of this package for your version of R might be available elsewhere,
2025-12-18T16:06:47.0333830Z see the ideas at
2025-12-18T16:06:47.0334401Z https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages 
```

**Comment by ThierryO** [comment]
You can add dependencies to the `checklist.yml`. That is not required for packages on CRAN. Or listed in the additonal repositories in `DESCRIPTION`.

```
x <- read_checklist()
x$set_pak("STBrinkmann/GVI")
write_checklist(x)
```

Note that you need the devel version of checklist.

**Comment by wlangera** [comment]
> I think it has been improved, but there is still some issue. [Here ](https://github.com/inbo/mbag-mas/actions/runs/20434106615/job/58711247690?pr=52) are the new logs.
> 
> ```
> 2025-12-22T14:07:23.9101707Z ✔ Installed GVI 1.2 (github::STBrinkmann/GVI@999c7f0) (52ms)
> 2025-12-22T14:07:23.9253344Z ✔ 1 pkg + 49 deps: kept 40, added 2, dld 2 (NA B) [2m 28.8s]
> 2025-12-22T14:07:30.9595373Z Error: (converted from warning) unable to access index for repository https://inla.r-inla-download.org/R/stable/src/contrib:
> 2025-12-22T14:07:30.9596026Z   cannot open URL 'https://inla.r-inla-download.org/R/stable/src/contrib/PACKAGES'
> 2025-12-22T14:07:30.9596374Z Execution halted
> 2025-12-22T14:07:30.9710378Z 
> 2025-12-22T14:07:30.9711075Z Failed to install missing packages. Please check the error message above.
> ```

Nevermind. It is fixed. Thanks
